app/controllers/users_controller.rb

```diff
@@ -9,7 +9,11 @@ class UsersController < ApplicationController
   # GET /users
   # GET /users.json
   def index
-    @users = User.paginate(page: params[:page], per_page: 10)
+    if admin_user
```

Comment:
Should this be changed to:
admin_user
@users = User.paginate(page: params[:page], per_page: 10)

and admin_user is changed to -> see admin_user